apps/aecore/src/aec_accounts.erl

```diff
@@ -0,0 +1,57 @@
+-module(aec_accounts).
+
+%% API
+-export([empty/0,
+         get/2,
+         get_with_proof/2,
+         put/2,
+         earn/3,
+         root_hash/1]).
+
+-include("trees.hrl").
+
+empty() ->
+    {ok, gb_merkle_trees:empty()}.
+
+get(Pubkey, AccountsTree) ->
+    case gb_merkle_trees:lookup(Pubkey, AccountsTree) of
```

Comment:
Do we want to hide calls to `gb_merkle_trees` in `trees.erl`, so that we have Erlang APIs to operate on tree library, and we don't have to modify N modules when we'll try to change tree library in the future?
I propose:
* `trees:lookup(Key, Tree)` / `trees:get(Key, Trees)`
* `trees:merkle_proof(Key, Tree)`
(...)
Wdyt?

Reply:
I agree. I will do this.